mapswipe_workers/sample_data/test_invalid_results.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Is there an actual test attached to this data, or is it just a record of what broken data looks like? It would be worth adding a comment in the JSON to explain what it means, otherwise it looks a lot like a random file committed by mistake :)

Reply:
we don't have a test for this, I just used it to manually upload it to firebase to test it in dev? rather delete then?

Reply:
I would keep the file, it's useful. Maybe add a line in the json like below since json doesn't really support comments. Ideally explain briefly how to use it to test, because in 3 months, we'll all have forgotten:
```json
  // your other lines
{
  "comment": "this is for...."
}
```